util/topgen.py

```diff
@@ -475,15 +469,10 @@ def generate_top_only(top_only_dict: Dict[str, bool], out_path: Path,
     log.info("Generating top only modules")
 
     for ip, reggen_only in top_only_dict.items():
-
-        if reggen_only and alt_hjson_path is not None:
-            hjson_dir = Path(alt_hjson_path)
-        else:
-            hjson_dir = (SRCTREE_TOP / "hw" / top_name / "ip" / ip / "data")
-
-        hjson_path = hjson_dir / f"{ip}.hjson"
-
-        genrtl_dir = out_path / "ip" / ip / "rtl"
+        log.info(f"generate_top_only: Ip {ip}, reggen {reggen_only} alt_hjson {alt_hjson_path}")
```

Comment:
I think we're no longer looking at `alt_hjson_path`. Is that intentional? (If so, it can probably be ripped out?)

Reply:
It is intentional. There is no topgen invocation that uses it, and if someone really wanted to do it they can just overwrite the hjson with their alternate.